15LessonFifteen/Checkpoint3/index.html

```diff
@@ -1,10 +1,113 @@
+
+  </ul>
+
+  <form class="navbar-form navbar-left">
+    <div class="form-group">
+      <input class="form-control" placeholder="Search" type="text">
+    </div>
+  <button class="btn">Submit</button>
+  </form>
+
+  <ul class="nav navbar-nav navbar-right">
+  <li><a href="#">Sign In</a></li>
+  <li><a href="#">New User</a></li>
+  </ul>
+
+  </div>
+  </nav>
+
+  <section class="hero col-lg-12">
+    <div class="herotext col-lg-4 col-lg-offset-4">
+      <img class="img-responsive" src="http://foundry.mediumra.re/img/wine1.png"/>
+      <h3>"Without doubt, one of the most consistently high-quality wineries in the Barossa region"</h3>
+      <p>David Halladay - Respected Wino</p>
+    </div>
+  </section>
+
+<section class="washer col-lg-12">
+  <div class="washertext col-xs-6">
+    <p>EST. 1968</p>
+    <h2>Combining over 100 years of hard work and shared family knowledge</h2>
+    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+  </div>
+  <div class="washerimage col-xs-6">
+    <img class="img-responsive" src="./img/wine16.jpg"/>
+  </div>
+</section>
+
+<section class="producing col-lg-12">
+<div class="producingtext col-lg-6 col-lg-offset-3">
+  <h3>We've been producing fine wine for over 65 years, I think we know what we're talking about</h3>
+  <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+</div>
+</section>
+
+<section class="types col-1g-12">
+  <div class="typesshiraz col-lg-4">
+    <img class="img-responsive" src="./img/wine3.jpg"/> <hr>
+    <h4>Foundry Shiraz</h4>
+    <p>“Aged five years in solid, French Oak barrels sports notes of pepper and blackberry”</p>
+    <a href="#">Visit Shop</a>
+  </div>
+  <div class="typesshiraz col-lg-4">
+    <img class="img-responsive" src="./img/wine6.jpg"/><hr>
+    <h4>Foundry Chardonnay</h4>
+    <p>“A rich fruit palet with subtle Oak influence and a lingering dry, fresh finish”</p>
+    <a href="#">Visit Shop</a>
+  </div>
+  <div class="typesshiraz col-lg-4">
+    <img class="img-responsive" src="./img/wine7.jpg"/><hr>
+    <h4>Foundry Cabernnay</h4>
+    <p>“Generous sweet blackcurrant and mulberry character with soft, velvety tannins”</p>
+    <a href="#">Visit Shop</a>
+  </div>
+</section>
+
+
+
+
+
+
+
+  <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+  <script src="https://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+
+  <!-- Latest compiled and minified JavaScript -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
 
 </body>
 </html>
```

Comment:
overall, this looks good, but your code has some inconsistencies between using bootstrap and plain html. because these are maintained in different places, it can be more difficult and time-consuming to debug